op-e2e/eip4844_test.go

```diff
@@ -86,6 +87,9 @@ func testSystem4844E2E(t *testing.T, multiBlob bool) {
 	opts.Value = mintAmount
 	SendDepositTx(t, cfg, l1Client, l2Verif, opts, func(l2Opts *DepositTxOpts) {})
 
+	// block the txpool to test the tx clearing functionality.
+	sys.BatchSubmitter.Driver().JamTxPool(context.Background())
+
```

Comment:
This response to jamming is the thing that changes with the fix, right? We could commit just this Jam call and its definition in order to demonstrate the issue?

I feel iffy about having a chaos-monkey function exposed on the TXManager.